xdsl/irdl/declarative_assembly_format.py

```diff
@@ -159,11 +161,13 @@ def parse(
             properties = state.properties
         else:
             properties = op_def.split_properties(state.attributes)
+
```

Comment:
```suggestion
```

Reply:
I feel like it's easier just to post suggestions then it's two clicks in the GH UI to apply them

Reply:
I just forgot this was a feature